client/app/helpers/format-date.coffee

```diff
@@ -1,10 +1,11 @@
 `import Ember from 'ember'`
 
-formatDate = (date, options) ->
+formatDate = (date, format) ->
```

Comment:
dug into it a bit:
If you explicitly pass a second argument, that second argument will be what you pass it. If you don't, it is NOT undefined, instead there is a default options hash like `Object {hash: Object, data: Object}`. Will think about the best way to do this since an ideal API for this helper should be `{{formatDate blahDate 'LLL'}}}`